src/frontend/src/common/utils/userInfo.ts

```diff
@@ -0,0 +1,21 @@
+import { find } from "lodash";
+import { store } from "../redux";
+import { selectCurrentGroup } from "../redux/selectors";
+
+/**
+ * Takes the stored id for the current group and returns the available details about that group
+ * from /me response
+ */
+export const getCurrentGroupFromUserInfo = (
+  userInfo?: User
+): UserGroup | undefined => {
+  const state = store.getState();
+  const currentGroupId = selectCurrentGroup(state);
+  return find(userInfo?.groups, (g) => g.id === currentGroupId);
+};
```

Comment:
do we need a fallback for if userInfo is undefined? Or do we not need to worry about that anymore? I remember that causing some strange side effects before

Reply:
It caused strange side effects in the past because we were using that -1 group id to call the api, which we aren't doing anymore. we shouldn't need a fallback here.